ROADMAP.md

```diff
@@ -90,16 +90,58 @@ See details in [GitHub milestone v1.3.0](https://github.com/ratify-project/ratif
 
 See details in [GitHub milestone v1.4.0](https://github.com/ratify-project/ratify/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.4.0).
 
-### v2.x
+### v2.0.0-alpha.1
+**Status**: In procoess
+**Target date**: May 15, 2025
+
+**Major features**
+
+- Decouple Ratify core functionalities into a separate library([ratify-go](https://github.com/ratify-project/ratify-go)), which could be easily integrated into different projects.
+- Design new interfaces for `Verifier`, `Store`, and `PolicyEnforcer` to make them more extensible and flexible.
+- Support `notation-verifier` under [ratify-verifier-go](https://github.com/ratify-project/ratify-verifier-go) in terms of the new `Verifier` interface.
+- Support Registry Store and local OCI layout in terms of the new `Store` interface.
+- Support config based policy in terms of the new `PolicyEnforcer` interface.
+- Create new external data provider for Gatekeeper based on the new framework.
+
+### v2.0.0-alpha.2
+**Status**: Tentative
+**Target date**: June 30, 2025
+**Major features**
+- Support authentication with remote OCI registries.
+- Support key/certificate management for verifiers.
+
+### v2.0.0-beta.1
+**Status**: Tentative
+**Target date**: TBD
+**Major features**
+- Support cache management.
```

Comment:
Just wondering if Ratify cache still relies on Redis+Dapr in v2?

Reply:
It's not finalized yet. Ratify v1 has filesystem, in-memory and redis caches, which would be also considered in v2. Redis+Dapr is just an option for cluster. And based on the feedback from v1 users, `Dapr` would be removed to reduce dependency and manual configuration steps.